Tasks/AzureRmWebAppDeployment/LegacyUtils/XdtTransformation-Legacy.ps1

```diff
@@ -0,0 +1,41 @@
+$XmlTranformDllPath = "$PSScriptRoot\Microsoft.Web.Xdt\Microsoft.Web.XmlTransform.dll"
```

Comment:
Without this transformation classes won't be available right? Hence loading this